src/Pages/SignIn/SignIn.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import MemberSignIn from "./MemberSignIn";
+import NonMemberSignIn from "./NonMemberSignIn";
+import "./SignIn.scss";
+
+class SignIn extends Component {
+  constructor() {
+    super();
+    this.state = {
+      account: "",
+      password: "",
+      activeID: 0,
+    };
+  }
+
+  handleInput = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  tabHandler = (id) => {
+    console.log(id);
+    this.setState({ activeID: id });
+  };
+
+  handleLoginBtn = () => {
+    fetch("http://10.58.3.224:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        account: this.state.account,
+        password: this.state.password,
+      }),
+    })
```

Comment:
```suggestion
  handleLoginBtn = () => {
    const { account, password } = this.state
    fetch("http://10.58.3.224:8000/user/signin", {
      method: "POST",
      body: JSON.stringify({ account, password }),
    })
```
1. 구조분해 할당을 사용하면 this.state라고 반복되는 부분을 생략할 수 있습니다
2. 만약 객체의 키값과 value 자리의 변수명이 동일하다면 생략할 수 있습니다. 객체 단축 속성명이라고 합니다.